scripts/dc2/io_utils.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""Utility functions for reading and writing tables
+
+"""
+
+import os
+import numpy as np
+import sqlite3
+from sqlalchemy import create_engine
+from astropy.io import fits
+import pandas as pd
+
+__all__ = ['to_csv', 'export_db']
+
+def to_csv(truth_db_path, dest_dir='.', table_suffix=''):
+    """Dumps sqlite3 files of the truth tables as csv files
+
+    Parameters
+    ----------
+    truth_db_path : str
+        Path to the truth tables Bryce made, either of the lens or the host
+
+    """
+    db = sqlite3.connect(truth_db_path)
+    cursor = db.cursor()
+    cursor.execute("SELECT name FROM sqlite_master WHERE type='table';")
+    tables = cursor.fetchall()
+    for table_name in tables:
+        table_name = table_name[0]
+        table = pd.read_sql_query("SELECT * from %s" % table_name, db)
+        table.to_csv(os.path.join(dest_dir, table_name + '{:s}.csv'.format(table_suffix)), index=False)
+    cursor.close()
+    db.close()
+
+def export_db(dataframe, out_dir, out_fname, table_name, overwrite=False):
+    """Export a DB from a Pandas DataFrame
+
+    Parameters
+    ----------
+    dataframe : Pandas.DataFrame object
+    out_fname : str
+    table_name : str
+    overwrite_existing : bool
+    
+    """
+    out_path = os.path.join(out_dir, out_fname)
+    if overwrite is True and os.path.exists(out_path):
+        os.remove(out_path)
+    engine = create_engine('sqlite:///{:s}'.format(out_path), echo=False)
+    dataframe.to_sql(table_name, con=engine, index=False)
```

Comment:
I think we need to add
```
    engine.dispose()
```
after this line to ensure that the the sqlite3 file gets written completely.  I've had some incomplete files produced when I've run this code.